src/main/java/buildcraftAdditions/items/Tools/ItemPipeColoringTool.java

```diff
@@ -33,9 +36,11 @@
  * Please check the contents of the license located in
  * http://buildcraftadditions.wordpress.com/wiki/licensing-stuff/
  */
-public class ItemPipeColoringTool extends Item {
+public class ItemPipeColoringTool extends Item implements IEnergyContainerItem {
```

Comment:
Its simply copying from ItemEnergyContainer... I am working on this configurable updates thing and @BigXplosion has this PR open. I would say @BigXplosion should do this.